doc/server/tutorial.rst

```diff
@@ -216,6 +216,46 @@ assignments that do not change the value will still be reported). This default
 can be changed by passing `auto_strategy` and `auto_strategy_parameters` when
 constructing the sensor.
 
+Aggregate Sensors
+-----------------
+To provide a sensor which has its reading derived from that of a set of other
+sensors, such as a total, average or general "device status" sensor, the
+:class:`.AggregateSensor` class allows the user to implement this. The user
+should subclass :class:`.AggregateSensor`, and implement
+:func:`.~AggregateSensor.update_aggregate` in order to define the logic which will
+be used.
```

Comment:
The clause after "To provide" should probably start with a verb, and this can generally be made less wordy e.g.
```suggestion
To provide a sensor which has its reading derived from that of a set of other
sensors, such as a total, average or general "device status" sensor, subclass
:class:`.AggregateSensor`, and implement
:func:`.~AggregateSensor.update_aggregate` to compute the sensor reading from
changes in sensor values.
```